src/main/java/DialogBox.java

```diff
@@ -0,0 +1,59 @@
+import java.io.IOException;
```

Comment:
I think generally your code is very clean and well-written, just that maybe not all your methods have JavaDocs, but still very readable. Also one thing I liked was the Japanese chatbot haha